permissions/plugin-sideex.yml

```
@@ -0,0 +1,7 @@
+---
+name: "sideex"
+github: "jenkinsci/sideex-plugin"
+paths:
+- "org/jenkins-ci/plugins/sideex"
```

```suggestion
- "io/jenkinsi/plugins/sideex"
```
Should be the groupId + artifactId, your groupId in your pom.xml is io.jenkins.plugins.